wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -4,11 +4,13 @@
 {-# LANGUAGE LambdaCase                 #-}
 {-# LANGUAGE OverloadedStrings          #-}
 {-# LANGUAGE TemplateHaskell            #-}
+{-# LANGUAGE StandaloneDeriving         #-}
```

Comment:
Why do we need `StandaloneDeriving` here? If that's just for `Generic` deriving, cannot we use `DeriveGeneric`?